tests_zemu/tests/basic.spec.js

```diff
@@ -267,14 +272,18 @@ describe("Basic checks", function () {
 
             const scheme = FlowApp.Signature.SECP256K1 | FlowApp.Hash.SHA2_256;
             const path = `m/44'/539'/${scheme}'/0/0`;
-            const resp = await app.getAddressAndPubKey(path);
+            const address = "0123456789abcdef"
+
+            await prepareSlot(sim, app, 0, address, path)
```

Comment:
After making the update above, we should be checking the output of `prepareSlot` as part of the tests.